src/Client.js

```
@@ -584,6 +584,20 @@ class Client extends EventEmitter {
         return chatId._serialized;
     }
 
+    /**
+     * Accepts a private invitation to join a group
+     * @param {object} inviteInfo Invite V4 Info
+     * @returns {Promise<Object>}
+     */
+     async acceptGroupV4Invite(inviteInfo) {
+        if(!inviteInfo.inviteCodeV4) return 'Invalid invite code'
+        if (inviteInfo.inviteCodeExpV4 == 0) return 'Expired invite code'
```

We should `throw` here instead of returning to indicate something went wrong